### PR TITLE
Messages (part 2)

### PR DESCRIPTION
**Pull request by wismill**
Follow-up of #332 

- Merge log functions with and without message code.
- Add more messages entries.

### CODE REVIEW

File: src/messages-codes.h.jinja [hunk 1]
```diff
@@ -6,6 +6,14 @@
 
 #include <stdint.h>
 
+/**
+ * Special case when no message identifier is defined.
+ *
+ * @added 1.6.0
```
Comment (bluetech) [inline src/messages-codes.h.jinja:12]: This is not public API so the `@added` is not needed

**Review by bluetech — APPROVED** [review]
Excellent work. Everything looks good to me.

File: src/messages-codes.h.jinja (hunk 1 above)
Comment (wismill) [reply src/messages-codes.h.jinja]: Right! I also removed it from `enum xkb_message_code`.

**Comment by wismill** [comment]
Applied review fixes and rework the commit messages.

**Pull request merged**